content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md

```diff
@@ -30,10 +30,12 @@ For more detailed information, see [Receiving update notifications](https://docu
 
 The input blocks can now receive inputs from all input sources by default. You can now simplify global tasks by creating a single Analytics Builder model that works with inputs from all input sources, for example, send an email for every critical alarm of type `C8Y_TemperatureAlarm` that is generated. A new **All Inputs** option is available for this purpose.
 When you edit the parameters of a newly added input block, the **All Inputs** option is now set by default. See also [Editing the parameters of a block](https://cumulocity.com/docs/streaming-analytics/analytics-builder/#editing-the-parameters-of-a-block) in the user documentation.
-You can also use the new **All Inputs** option to replace all input sources. Keep in mind that the replace dialog box is used for both input blocks and output blocks. So when you replace a device with the new **All Inputs** option, all matching output devices are automatically changed to trigger devices.
 
+You can also use the new **All Inputs** option to replace all input sources. Keep in mind that the replace dialog box is used for both input blocks and output blocks. So when you replace a device with the new **All Inputs** option, all matching output devices are automatically changed to trigger devices.
 See also [Replacing sources or destinations](https://cumulocity.com/docs/streaming-analytics/analytics-builder/#replacing-sources-or-destinations) in the user documentation.
 
+A new sample named "Aggregate measurements per input source" is now available, which creates new measurements that average the measurement values for each input source that has a specified fragment and series. This is a simple sample without template parameters, so you can activate it directly in the model manager. See also [The Samples tab](https://cumulocity.com/docs/streaming-analytics/analytics-builder/#the-samples-tab) in the user documentation.
+
```

Comment:
Should we be saying that this sample **creates** a model without template parameters...

Reply:
Right, that's better. Updated the doc, together with the changes from Rob.